scripts/pretty-printers/gdb/pretty_printers.py

```diff
@@ -13,7 +13,8 @@
         pretty_printer_options = json.load(json_file)
 except JSONDecodeError as e:
     print("Options file at {0} failed to load. Exception: {1}".format(options_path, e.msg))
-
+except Exception as e:
```

Comment:
:question: err -- maybe print the error? is there a reason to expect errors that can be ignored?

Reply:
Yes - an options file isn't integral, so if anything goes wrong in the block, even on the import statements, we don't want it to stop the pretty-printer. Might print an error though, just so it isn't silently failing for a long time.